plugin/hooks.go

```diff
@@ -288,7 +288,7 @@ type Hooks interface {
 	// EXPERIMENTAL: This hook is experimental and can be changed without advance notice.
 	//
 	// Minimum server version: 7.6
-	UserHasPermissionToCollection(c *Context, user *model.User, collectionType, collectionId string, permission *model.Permission) (bool, error)
+	UserHasPermissionToCollection(c *Context, userId string, collectionType, collectionId string, permission *model.Permission) (bool, error)
```

Comment:
```suggestion
	UserHasPermissionToCollection(c *Context, userId, collectionType, collectionId string, permission *model.Permission) (bool, error)
```

Reply:
0/5 to have consistency of `userID` vs `userId`